src/api/core/operators/test_detect_text_in_image_tesseract.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import detect_text_in_image_tesseract
+
+class Test(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # initialize operator
+        detect_text_in_image_tesseract.initialize(param={})
+    
+    @classmethod
+    def tearDownClass(cls):
+        # delete config files
+        pass
+
+    def test_sample_image_from_disk(self):
+        image_path = "sample_data/text.png"
```

Comment:
@aatmanvaidya looks like the text depends on a file stored on disk? Can we add this to the repo too? you can create the `sample_data` folder in the `operators` folder itself.

Reply:
so in the other test files, the `sample_data` folder is just refereed, and it was also there in the .gitignore file, that's why I didn't add it, but I can add it now

Reply:
got it. yeah I think with just me working on it, that was fine but lets include the sample_data folder to the repo to reduce the steps to setup feluda locally. As long as we are careful about not uploading a large 10 gb video file in it, we should be fine.